src/main/resources/templates/success.html

```diff
@@ -3,15 +3,18 @@
 <head>
   <meta charset="UTF-8">
   <title>Trusona Gateway Response</title>
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
 </head>
-<body>
+<body style="text-align:left; margin-left:10em;margin-top:2em;font-family:'Lato',sans-serif;">
 
-<dl>
+<table width="50%">
```

Comment:
Booo, what was wrong with using `<dl>`?

Reply:
it was ugly AF

Reply:
So your solution was to slap it in a table?

Reply:
Was there some acceptance criteria for this change?

Reply:
no. this is an internal tool for testing.
it's either we keep the `dl` tag and have ugly output, or semi-decent formatting on the output